virt/virt-2-6-release-notes.adoc

```diff
@@ -106,9 +95,9 @@ As a workaround, you can reconfigure the virtual machines so that they can be po
 . Remove the `evictionStrategy: LiveMigrate` field. See xref:../virt/live_migration/virt-configuring-vmi-eviction-strategy.adoc#virt-configuring-vmi-eviction-strategy[Configuring virtual machine eviction strategy] for more information on how to configure eviction strategy.
 . Set the `runStrategy` field to `Always`.
 
-* For unknown reasons, memory consumption for the `containerDisk` volume type might gradually increase until it exceeds the memory limit. To resolve this issue, restart the VM. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1855067[*BZ#1855067*])
+//Is this still relevant with the UI changes (the bug itself has not been fixed)? If it is still relevant, how should we change the patch command to reflect the 2.6 version?
 
-* Sometimes, when attempting to edit the subscription channel of the *{VirtProductName} Operator* in the web console, clicking the *Channel* button of the *Subscription Overview* results in a JavaScript error. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1796410[*BZ#1796410*])
+* Sometimes, when attempting to edit the subscription channel of the {VirtProductName} Operator in the web console, clicking the *Channel* button of the *Subscription Overview* results in a JavaScript error. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1796410[*BZ#1796410*])
```

Comment:
@tiraboschi Do you know if this known issue should be retained for 2.6 (see comment)?

Reply:
https://bugzilla.redhat.com/1796410 is an OCP UI bug and it's still on assigned, so yes, we should keep the note.

Reply:
@tiraboschi, please check the changes to the patch command below the note when you get the chance. Thanks!